### ISSUE TITLE
Support React v15 as peer dependency

### ISSUE THREAD

**Issue by albertogasparin** [WickyNilliams/react-simple-colorpicker#14]
Your plugin is working fine, just need to add:

```json
"peerDependencies": {
  "react": "^0.14.0 || ^15.0.1",
  "react-dom": "^0.14.0 || ^15.0.1"
}
```

**Comment by jonnywyatt** [comment]
Yes please support this as it's failing npm shrinkwrap

**Comment by WickyNilliams** [comment]
Fixed in 50e1ee4

**Comment by WickyNilliams** [comment]
(and published to npm as v1.4.0)
